examples/nextjs-live-avatars/components/Avatar.tsx

```diff
@@ -12,8 +12,15 @@ import styles from "./Avatar.module.css";
  */
 
 const IMAGE_SIZE = 48;
+//convert to typescript
+export interface AvatarProps {
```

Comment:
Should be renamed to `Props` because we're in a single file component. This is a common convention.